### DISCUSSION TITLE
opencascade breaks building qemu

### DISCUSSION THREAD

**Discussion by revelator** [msys2/MINGW-packages#16304] [Q&A] [answered]
this one had me baffled a bit, the env.bat from opencascade is creating havoc with some sources using env "someexecutable.exe"
one example being some python tools using env python end up using the batch file instead of the msys2 env.exe...
this took me some hunting because i was unable to build qemu myself due to sphinx throwing a fit untill i noticed that it was using the env.bat file Oo.

easy fix if reported upstream would be to simply rename the batch file to say setenv.bat and make draw.bat reference that one instead so far i had no problems with it, so is this something that could be fixed as i have a nagging feeling it may also interfere with other scripts using msys2's env.exe ?.

**Comment by Biswa96** [comment]
What errors are you seeing? How to reproduce it?

**Comment by revelator** [comment]
it spews an error about non existing etc//"" in sphinx document creation if opencascade is installed, this is caused by sphinx using opencascades env.bat instead of msys2's env.exe.

**Reply by Biswa96** [reply]
Agree. I can reproduce the issue. Next time, please open a proper issue if it is reproducible in CI or in clean install. There were some cases where reinstalling solved your issue.

**Reply by revelator** [reply]
allways a good idea to discuss problems before reporting them i think :) you might have come up with a better fix than mine.
atleast it is reproducible by you. indeed reinstalling has fixed a lot but id rather not have to do that everytime something odd happens, in some cases like this it is just a simple name clash and easy to fix :)

**Comment by Biswa96** [comment, marked as answer]
This pull request may fix the issue https://github.com/msys2/MINGW-packages/pull/16340

**Reply by revelator** [reply]
guess the batch files only really mattered for msvc :) so removing them seems to be prudent.

**Reply by Biswa96** [reply]
The fixed opencascade package version 7.7.0-6 is in the pacman repository. Now I can build qemu in presence of opencascade package. Thank you for reporting the issue.
